entry.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+db_migrate=true
+
+while true; do
+  case "$1" in
+    --no-migrate)
+      db_migrate=false
+      ;;
+    --)
```

Comment:
What are we matching here?

Reply:
standard command line stuff: `command [--no-migrate] [--] ...`

`--` is often signaling end of CLI flags. it can be useful if you wanted to start a command starting with `--` in your container. Unlikely but this is just like escaping variabls with spaces in shell scripts. Better always do it so you don't have edge cases.

Reply:
It's like in grep. If you want to match a pattern starting with `-` (`-foo-` for example` you will be calling `grep -- -foo- FILE`

Reply:
Cool. Thanks for sharing! :)